jstz_node/src/services/accounts.rs

```diff
@@ -29,11 +39,60 @@ async fn nonce(
     Ok(HttpResponse::Ok().json(nonce))
 }
 
+#[get("/{address}/kv")]
+async fn kv(
+    rollup_client: Data<RollupClient>,
+    path: Path<String>,
+    query: web::Query<HashMap<String, String>>,
+) -> Result<impl Responder> {
+    let address = path.into_inner();
+    let key_option = query.get("key").cloned();
```

Comment:
Nit: `.as_ref()` provides a `Option<&str>` which would avoid some clones
NB: Don't commit suggestion, other changes are required for this

```suggestion
    let key_option = query.get("key").as_ref();
```